frontend/src/utils/chart/chart-style.js

```diff
@@ -0,0 +1,93 @@
+export const popupFormatter = (params) => {
+  var value = (params.value + "").split(".");
+  value = value[0].replace(/(\d{1,3})(?=(?:\d{3})+(?!\d))/g, "$1,");
+  if (Number.isNaN(params.value)) {
+    return;
+  }
+  return params.name + ": " + value;
+};
+
+export const backgroundColor = {
+  backgroundColor: "transparent",
+};
+
+export const Easing = {
+  animation: true,
+  animationThreshold: 2000,
+  animationDuration: 1000,
+  animationEasing: "cubicOut",
+  animationDelay: 0,
+  animationDurationUpdate: 300,
+  animationEasingUpdate: "cubicOut",
+  animationDelayUpdate: 0,
+};
+
+export const TextStyle = {
+  textStyle: {
+    color: "#000",
+    fontFamily: "Gotham A,Gotham B",
+    fontSize: "1rem",
+    fontWeight: "bold",
+  },
+};
+
+export const Color = {
+  color: [
+    "#36aa40",
+    "#0081e0",
+    "#2872c6",
+    "#355c7d",
+    "#3D588A",
+    "#5C616A",
+    "#C9CDC0",
+    "#DBC2CF",
+    "#2E4756",
+    "#CC9485",
+    "#CCD3B6",
+    "#FFE800",
+  ],
+};
+
+export const visualMap = {
+  left: "right",
+  top: "top",
+  inRange: {
+    color: ["#f4f7b5", "#1890ff"],
+  },
+  itemHeight: "520px",
+  text: ["High", "Low"],
+  calculable: true,
+};
+
+export const rsrColor = {
+  color: ["#355c7d", "#6c5b7b", "#c06c84", "#f67280", "#f8b195", "#ddd"],
+};
+
+export const Legend = {
+  orient: "horizontal",
+  x: "center",
+  y: "bottom",
+  bottom: "40px",
+  textStyle: {
+    fontFamily: "Gotham A,Gotham B",
```

Comment:
I'm curious about how do we define/include this font 🤔 ?

Reply:
I'm sorry, 
That code coming from previous work on other project using e-charts, 
So we not using Gotham font, I will change that.